### DISCUSSION TITLE
nginx-proxy with bare-metal kubernates

### DISCUSSION THREAD

**Discussion by abneetwats24** [nginx-proxy/nginx-proxy#2051] [Q&A] [answered]
Can nginx-proxy be used with bare-metal Kubernetes cluster?

**Comment by coaperator** [comment, marked as answer]
What for? Kubernetes has its own tools for this, it seems ..

**Reply by abneetwats24** [reply]
Right got it, I was trying to use nginx-proxy as an ingress controller with the capability of VIRTuAL_HosT and HTTPS.
